<launch>
  <arg name="veh" doc="Name of vehicle. ex: megaman"/>
  <arg name="package_namespace" default="calibration" />
  <arg name="node_namespace" default="data_collector" />

  <!-- Launch the rosbag_recorder node to start/stop bag file inside a node in this package -->
  <node  pkg="rosbag_recorder" type="rosbag_recorder_server.py" name="rosbag_recorder_server" output="screen"/>

  <!-- include the machines file  -->
  <include file="$(find duckietown)/machines"/>

  <!-- launch the wheel drive node -->
  <include file="$(find dagu_car)/launch/wheels_driver_node.launch">
    <arg name="veh" value="$(arg veh)"/>
  </include>

  <!-- Launch the script that publishes open-loop input commands to wheels -->
  <group ns="$(arg veh)/$(arg package_namespace)">
    <node  pkg="calibration" type="$(arg node_namespace).py" name="$(arg node_namespace)" output="screen" clear_params="true" required="true">
      <param name="veh" value="$(arg veh)"/>
    </node>
  </group>
</launch>
